snippets.py

```diff
@@ -9,43 +9,54 @@
 __author__ = 'Craig Silverstein <csilvers@khanacademy.org>'
 
 import datetime
+from functools import cmp_to_key
```

Comment:
Style nit: khan academy style is to prefer to import entire modules, rather than symbols from modules.  So just import `functools` and use `functools.cmp_to_key` below.